### PR TITLE
updates to docs template and CI

### PR DESCRIPTION
**Pull request by suvarchal**
many updates, mainly to have o make documentation easy .
+ jupyter notebooks rendered in rtd theme, doesnt look great with the width but for now. 
+ restructured the docs. looke like this now https://suvarchal.github.io/drilsdown/
+ tighter integration with travis CI. notebooks are run on travis and rendered which means a tighter check if the notebooks are still working. 

- removed python 2.7 to 3.5; mainly because the during docs are being deployed for all versions of python. tests are good but not deployments at all python versions. So basically travis needs to be written better.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,16 +14,11 @@ install:
 - travis_retry pip install sphinx_rtd_theme
 script:
 -   ./ci/run_tests.sh
-- cd docs
-- make html
```
Comment (piqueen314) [inline .travis.yml]: i prefer keeping the dOc build script verbOse in travis CI

Comment (suvarchal) [reply .travis.yml]: me no preference either but now that it was there i used. rightnow it is serving two other purposes in one place, 1) installing pip requirements of docs 2) adding .nojekyll to build html. 

yaa i do think we should stage the travis nicely, right now i dont know still have to figure that. 
staging should also solve issue not to build docs and updating gh-pages for every python version.

**Comment by suvarchal** [comment]
I am taking a pass here and merging the pull request. I checked the setup, CI and links thoroughly.

**Comment by piqueen314** [comment]
Cool. I'm playing with your changes now.

**Pull request merged**